Jenkinsfile

```
@@ -0,0 +1,30 @@
+#!groovy
+
+pipeline {
+  agent none
+
+  stages {
+    stage('build') {
+      when { branch 'main' }
+      agent { label 'ecs-builder-node14' }
+      steps {
+        initBuild()
+        sh 'yarn'
```

```suggestion
        frozenYarnInstall()
```
This will make sure Jenkins actually builds the versions you tested in dev.